src/agents/coordinator.ts

```diff
@@ -57,18 +61,23 @@ export class DkgCoordinatorAgent {
   }
 
   public static async initializeRitual(
-    provider: ethers.providers.Provider,
-    signer: ethers.Signer,
-    providers: ChecksumAddress[]
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _provider: ethers.providers.Provider,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _signer: ethers.Signer,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _providers: ChecksumAddress[]
   ): Promise<number> {
-    const Coordinator = await this.connectReadWrite(provider, signer);
-    const tx = await Coordinator.initiateRitual(providers);
-    const txReceipt = await tx.wait(DEFAULT_WAIT_N_CONFIRMATIONS);
-    const [ritualStartEvent] = txReceipt.events ?? [];
-    if (!ritualStartEvent) {
-      throw new Error('Ritual start event not found');
-    }
-    return ritualStartEvent.args?.ritualId;
+    throw new Error('Not implemented');
+
+    // const Coordinator = await this.connectReadWrite(provider, signer);
```

Comment:
I will update it in a minute. It's out of scope for 7.0.0 and irrelevant to this PR so it can wait.